lib/runtime/storage.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 ChainSafe Systems (ON) Corp.
+// This file is part of gossamer.
+//
+// The gossamer library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The gossamer library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the gossamer library. If not, see <http://www.gnu.org/licenses/>.
+
+package runtime
```

Comment:
it might make more sense to have this in the `dot/state` package, since that's where all the database related functionality is

Reply:
when I move this to `dot/state` I get an import cycle, any suggestions?
```
package github.com/ChainSafe/gossamer/lib/runtime
	imports github.com/ChainSafe/gossamer/dot/state
	imports github.com/ChainSafe/gossamer/lib/genesis
	imports github.com/ChainSafe/gossamer/lib/runtime: import cycle not allowed

```